ScreenManager.c

```diff
@@ -106,7 +106,7 @@ static void checkRecalculation(ScreenManager* this, double* oldTime, int* sortTi
    if (*rescan) {
       *oldTime = newTime;
       ProcessList_scan(pl, this->state->pauseProcessUpdate);
-      if (*sortTimeout == 0 || this->settings->treeView) {
+      if (!(this->state->pauseProcessUpdate) && (*sortTimeout == 0 || this->settings->treeView)) {
```

Comment:
No need for parens …

```suggestion
      if (!this->state->pauseProcessUpdate && (*sortTimeout == 0 || this->settings->treeView)) {
```

Reply:
ok, I didn't find a guidance on this in the styleguide. I actually checked :)